lib/CLI/PrettyPrint.hs

```diff
@@ -98,3 +99,12 @@ buildField (fk, field) = do
 buildDate :: UTCTime -> Builder
 buildDate time =
   "[" <> build (formatTime defaultTimeLocale "%F %T" time) <> "]"
+
+buildPathNotFound :: QualifiedPath Path -> Builder
+buildPathNotFound path = [int|s|Entry or directory not found at '#{path}'.|]
+
+buildEntryNotFound :: QualifiedPath EntryPath -> Builder
+buildEntryNotFound entryPath = [int|s|Entry not found at '#{entryPath}'.|]
+
+buildSamePath :: QualifiedPath Path -> Builder
+buildSamePath path = [int|s|'#{path}' and '#{path}' are the same path.|]
```

Comment:
These 3 functions can also be moved to `Coffer.PrettyPrint`

Reply:
Done